app/models/content_configuration.rb

```diff
@@ -33,19 +33,19 @@ class ContentConfiguration < Spree::Preferences::FileConfiguration
 
   # Main URLs
   preference :menu_1, :boolean, default: true
-  preference :menu_1_icon_name, :string, default: ""
+  preference :menu_1_icon_name, :string, default: "ofn-i_019-map-pin"
   preference :menu_2, :boolean, default: true
-  preference :menu_2_icon_name, :string, default: ""
+  preference :menu_2_icon_name, :string, default: "ofn-i_037-map"
   preference :menu_3, :boolean, default: true
-  preference :menu_3_icon_name, :string, default: ""
+  preference :menu_3_icon_name, :string, default: "ofn-i_036-producers"
   preference :menu_4, :boolean, default: true
-  preference :menu_4_icon_name, :string, default: ""
+  preference :menu_4_icon_name, :string, default: "ofn-i_035-groups"
```

Comment:
Great! The one last thing I think I'd add is some small comments as a reference in this file to give anyone looking at the code a bit of help understanding which is which, so they don't have to look through the rest of the codebase for hints, eg:

``` ruby
preference :menu_1, :boolean, default: true # Shops
preference :menu_1_icon_name, :string, default: "ofn-i_019-map-pin"
preference :menu_2, :boolean, default: true # Maps
preference :menu_2_icon_name, :string, default: "ofn-i_037-map"
preference :menu_3, :boolean, default: true # Producers
```

Reply:
I can do that but tbh I dont think it's the best approach.
They will change the en.yml entries for AUS at some point and they will not need to change this file. So I'd not link the two things as they may become outdated.
Also, on the other side, a dev changing this needs to know that they correspond to the en.yml file entries, right? :-)